### ISSUE TITLE
Extension is not working now.

### ISSUE THREAD

**Issue by anecdot11** [ncla/SteamInventoryExpert#10]
According to SIH group
"Today Steam changed something on their side that causes our features to work incorrect or not work at all."

sauce: http://steamcommunity.com/groups/SteamInventoryHelper#announcements/detail/606128947923742108

**Comment by ncla** [comment]
https://github.com/SteamDatabase/SteamTracking/commit/5ec377772d84853deba30d99cb3c5fa58ebab306

Everything `economy.js` related got refactored, and because SIH is heavily relying on Steam's functions, they will have to refactor as well. It will be a while.

**Comment by Floyd77** [comment]
Hi, are you planning to fix SIE? SIH was updated today to work with the latest Steam changes in their code: http://steamcommunity.com/groups/SteamInventoryHelper#announcements/detail/606128947935837137

Thank you.

**Comment by ncla** [comment]
If you are not concerned about the ads then you can download the SIH extension from Chrome app store yourself and load it locally.

**Comment by Floyd77** [comment]
I only wanted to know if this extension is still alive or discontinued/abandoned. I prefer this ad-free version but I will use SIH if there is no choice.

Thanks again.

**Comment by Momchoto** [comment]
@Floyd77 You can very easily remove the advertisements yourself, I can even help you if you want. But you'll have to enable developer mode, that means every time you launch Chrome, there will be an annoying pop up warning you it's on.

**Comment by ncla** [comment]
I mean I could theoretically keep this up to date, but I would need to write a script to download the extension automatically, beautify the code in the assets, and then cherry-pick the changes manually.

The guys behind SIH haven't done anything horrible with the extension yet (with exception being ad performance tracking and the ads them self). If you still want to be cautious, I will repeat what I said – download the extension yourself (with CRX download extension) and load it locally. You can remove ads probably with uBlock element pick tool, no need to touch the source-code.

My intention with this repository was not to keep this up to date, but rather just to make a back-up of SIH at the time when it was acquired by CSGOFast owners. Sure I did update it once which was a very simple fix because the developers didn't publish an update for a solid week, which caused a serious concern, but they eventually came through.

**Comment by Floyd77** [comment]
I am a Firefox user and SIE was working great with it until the latest changes in Steam's code. Latest SIH version does not work with Firefox, I downloaded the crx, extracted to a folder and tried to load manifest.json from about:debugging, but it gives an error.

I suppose I'll need to use Chrome with the new SIH. Thanks for your answers!

**Comment by Nonickn** [comment]
>>with exception being ad performance tracking and the ads them self)

Hi!

Could someone tell me please what exactly I should remove from manifest.json (and probably from html folder?) to get rid off of ads/tracking?

Thank you.

**Comment by Magellano** [comment]
In SIH has long been fixed. It is a pity that the author SIE still didn't. Besides that it is now overdue.

**Comment by its-dreamchaser** [comment]
> Could someone tell me please what exactly I should remove from manifest.json (and probably from html folder?) to get rid off of ads/tracking?

Hey.. Did you figure out how it's done? I'd like to know this too and would prefer to edit the CRX from the Chrome store & load it locally, rather than trusting the auto-updating extension.

**Comment by Momchoto** [comment]
@its-dreamchaser If you're not experienced with programming, I suggest you simply delete all fishy images from the "assets" folder. Be careful not to remove important ones though!

**Comment by Nonickn** [comment]
> Hey.. Did you figure out how it's done?

Not really, I just unpacked it and loaded in Chrome localy, it's now different ID so autoupdate shouldn't work.

I was hoping someone can name files and lines need be deleted, even Momchoto who said 
>I can even help you if you want.

but seems like it's not gonna happens.

**Comment by Momchoto** [comment]
It's gonna _happens_, don't worry. Let me wrap up what I'm doing and I'll see what I can do.

**Comment by Nonickn** [comment]
Will be nice:)

**Comment by Momchoto** [comment]
There, I downloaded the CRX file, loaded it up in developer mode and used the inspect tools to remove all possible ads I could find. I may have broken something, but it seemed to work fine. I'm no programmer.

https://www.dropbox.com/s/asd7b1ovxo51i5q/Steam-Inventory-Helper_v1.10.3.2.rar?dl=0

**Comment by its-dreamchaser** [comment]
Thanks so much, really appreciate it ; shall give it a try now.

**Comment by its-dreamchaser** [comment]
If in case the popup about developer mode extensions becomes too annoying, [here's a way](https://goo.gl/EN8RU8) to get rid of it by patching chrome.dll.

**Comment by Symbai** [comment]
How do I load your dropbox addon in Firefox? I tried using about debug but it says the addon is invalid.
